src/core/etl/src/Flow/ETL/PHP/Type/Native/ScalarType.php

```diff
@@ -7,31 +7,63 @@
 use Flow\ETL\Exception\InvalidArgumentException;
 use Flow\ETL\PHP\Type\Type;
 
-enum ScalarType : string implements NativeType
+final class ScalarType implements NativeType
 {
-    case boolean = 'boolean';
-    case float = 'float';
-    case integer = 'integer';
-    case string = 'string';
-
-    public static function fromString(string $value) : self
-    {
-        return match (\strtolower($value)) {
-            'integer' => self::integer,
-            'float', 'double' => self::float,
-            'string' => self::string,
-            'boolean' => self::boolean,
+    public const BOOLEAN = 'boolean';
+
+    public const FLOAT = 'float';
+
+    public const INTEGER = 'integer';
```

Comment:
we might need to introduce here int32 and int64 in order to easier transform flow entries into parquet/avro types

Reply:
Added new issue for this: https://github.com/flow-php/flow/issues/771